apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/V1/SessionPoolTests.TargetedSessionPoolTests.cs

```diff
@@ -76,7 +76,7 @@ private static TargetedSessionPool CreatePool(bool acquireSessionsImmediately)
                      WriteSessionsFraction = 0.2
                 };
                 var parent = new SessionPool(client, options);
-                return new TargetedSessionPool(parent, s_databaseName, acquireSessionsImmediately);
+                return new TargetedSessionPool(parent, new PartitionKey( s_databaseName, null), acquireSessionsImmediately);
```

Comment:
We should have a test with a set database role so that we confirm that's being propagated to the acquire session request.